Source/WebCore/platform/animation/Animation.cpp

```diff
@@ -105,28 +68,28 @@ bool Animation::animationsMatch(const Animation& other, bool matchProperties) co
     bool result = m_name.string == other.m_name.string
         && m_playState == other.m_playState
         && m_compositeOperation == other.m_compositeOperation
-        && m_playStateSet == other.m_playStateSet
         && m_iterationCount == other.m_iterationCount
         && m_delay == other.m_delay
         && m_duration == other.m_duration
         && *(m_timingFunction.get()) == *(other.m_timingFunction.get())
         && m_nameStyleScopeOrdinal == other.m_nameStyleScopeOrdinal
         && m_direction == other.m_direction
         && m_fillMode == other.m_fillMode
-        && m_delaySet == other.m_delaySet
-        && m_directionSet == other.m_directionSet
-        && m_durationSet == other.m_durationSet
-        && m_fillModeSet == other.m_fillModeSet
-        && m_iterationCountSet == other.m_iterationCountSet
-        && m_nameSet == other.m_nameSet
-        && m_timingFunctionSet == other.m_timingFunctionSet
-        && m_compositeOperationSet == other.m_compositeOperationSet
+        && isDelaySet() == other.isDelaySet()
+        && isDirectionSet() == other.isDirectionSet()
+        && isDurationSet() == other.isDurationSet()
+        && isFillModeSet() == other.isFillModeSet()
+        && isIterationCountSet() == other.isIterationCountSet()
+        && isNameSet() == other.isNameSet()
+        && isTimingFunctionSet() == other.isTimingFunctionSet()
+        && isCompositeOperationSet() == other.isCompositeOperationSet()
+        && isPlayStateSet() == other.isPlayStateSet()
```

Comment:
Can't you just compare the OptionSets?

Reply:
No, because we don't want to check for `isPropertySet()` (see below).